Sources_v3/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -50,12 +50,19 @@ public struct ChatClientConfig {
     /// refreshed.
     public var tokenProvider: TokenProvider?
     
+    /// Flag for setting a ChatClient instance in connection-less mode.
+    /// A connection-less client is not able to connect to websocket and will not
+    /// receive websocket events. It can still observe and mutate database.
+    /// This flag is automatically set to `true` for app extensions
+    public let isConnectionless: Bool
+    
```

Comment:
maybe `isClientInActiveMode` + "This flag is automatically set to `false` for app extensions" ?

Reply:
Or maybe we can create a `Mode` enum with `active` and `passive` values? 🤔